### PR TITLE
Refactored project + some bug fixes

### PR DESCRIPTION
**Pull request by danyoungday**
Refactored project so demo goes in use_cases/eluc. There is no data or model folder in the use_cases/eluc folder because there are no files there and git does not track empty folders. Will work on refactoring data/ and model/ folders in the future.

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # MVP Climate Change Demo
 
-This is a demo of the MVP Climate Change app. It allows users to select a location and year from a map of the world, see its land use composition, and prescribe or manually make changes to it and see the predicted ELUC (Emissions from Land Use Change) and amount of land changed. It is a simple Dash app.
+This is a demo of the MVP Climate Change app. It allows users to select a location and year from a map of the world, see its land use composition, and prescribe or manually make changes to it and see the predicted ELUC (Emissions from Land Use Change) and amount of land changed. It is a simple Dash app. The demo can be found online at [landuse.evoluion.ml](https://landuse.evolution.ml)
```
Comment (danyoungday) [inline demo/README.md:3]: Added the URL here so users can see the finished product.

File: data/gcb/conversion/codes.csv [hunk 2]
```diff
@@ -0,0 +1,247 @@
+English short name lower case,Alpha-2 code,Alpha-3 code,Numeric code,ISO 3166-2
```
Comment (danyoungday) [inline data/gcb/conversion/codes.csv:1]: This file is for some experimentation I was doing in forest.ipynb

File: use_cases/eluc/demo/README.md [hunk 3]
```diff
@@ -33,8 +33,8 @@ docker build -t eluc-demo .
 then the container can be run with:
 ```
 docker run \
--v PATH_TO_PROJECT/mvp/demo/data/processed:/usr/local/cognizant/eluc/data/processed:ro \
--v PATH_TO_PROJECT/mvp/demo/predictors:/usr/local/cognizant/eluc/predictors:ro \
+-v PATH_TO_PROJECT/mvp/use_cases/eluc/demo/data/processed:/usr/local/cognizant/eluc/data/processed:ro \
```
Comment (danyoungday) [inline use_cases/eluc/demo/README.md:36]: Updated path for Docker deployment

File: use_cases/eluc/demo/app/Predictor.py [hunk 4]
```diff
@@ -15,6 +15,11 @@ class Predictor(ABC):
 
     @abstractmethod
     def predict(self, input):
+        """
```
Comment (danyoungday) [inline use_cases/eluc/demo/app/Predictor.py:18]: Beginning of template for custom predictors

**Review by ofrancon — APPROVED** [review]
lgtm thanks

**Pull request merged**